test/check-application

```diff
@@ -1569,6 +1569,95 @@ class TestFiles(testlib.MachineCase):
+        b.click("button:contains('Edit permissions')")
+
+        # TODO: test for title contains file1,file2
+        b.wait_in_text("#edit-permissions-owner", "admin")
+        b.wait_in_text("#edit-permissions-group", "admin")
+        b.select_from_dropdown("#edit-permissions-owner-access", "read-only")
+        b.select_from_dropdown("#edit-permissions-group-access", "no-access")
+        b.select_from_dropdown("#edit-permissions-other-access", "no-access")
+        b.click(".pf-v5-c-modal-box button.pf-m-primary")
+        b.wait_not_present(".pf-v5-c-modal-box")
+        self.assertEqual(self.stat("%A", "/home/admin/multiple/filename1"), "-r--------")
+        self.assertEqual(self.stat("%A", "/home/admin/multiple/filename2"), "-r--------")
+
+        # No owner/group shown if ownership differs
+        m.execute("chown root: /home/admin/multiple/filename1")
+        b.wait_in_text("[data-item='filename1'] .item-owner", "root")
+
+        # HACK: the selected state contains a copy of files state so permissions information is not updated.
+        b.mouse("[data-item='filename1']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "contextmenu")
+        b.click("button:contains('Edit permissions')")
+
+        b.wait_visible("#edit-permissions-owner-access")
+        b.wait_not_present("#edit-permissions-owner")
+        b.wait_not_present("#edit-permissions-group")
+        b.click(".pf-v5-c-modal-box button.pf-m-link")
+        b.wait_not_present(".pf-v5-c-modal-box")
+
+        m.execute("chown admin:root /home/admin/multiple/filename1")
+        b.wait_in_text("[data-item='filename1'] .item-owner", "admin:root")
+        # HACK: the selected state contains a copy of files state so permissions information is not updated.
+        b.mouse("[data-item='filename1']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "contextmenu")
+        b.click("button:contains('Edit permissions')")
+
+        b.wait_visible("#edit-permissions-owner-access")
+        b.wait_not_present("#edit-permissions-owner")
+        b.wait_not_present("#edit-permissions-group")
+        b.click(".pf-v5-c-modal-box button.pf-m-link")
+        b.wait_not_present(".pf-v5-c-modal-box")
+
+        # No Edit permissions when selecting a folder and files
+        b.mouse("[data-item='testdir']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "contextmenu")
+        b.wait_visible("button:contains('Delete')")
+        b.wait_not_present("button:contains('Edit permissions')")
+
+        # Editing > 10 files
+        b.mouse("[data-item='testdir']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "click", ctrlKey=True)
+        for i in range(3, 15):
+            b.mouse(f"[data-item='filename{i}']", "click", ctrlKey=True)
```

Comment:
Curious: why the filename2 hole? To avoid equalizing the permissions? adding a comment is appreciated